_targets.R

```diff
@@ -10,6 +10,7 @@ source("3_visualize.R")
 dir.create("1_fetch/out/", showWarnings = FALSE)
 dir.create("2_process/out/", showWarnings = FALSE)
 dir.create("3_visualize/out/", showWarnings = FALSE)
+dir.create("3_visualize/log/", showWarnings = FALSE)
```

Comment:
we would need to add 
```suggestion
dir.create("1_visualize/log/", showWarnings = FALSE)
dir.create("2_visualize/log/", showWarnings = FALSE)
dir.create("3_visualize/log/", showWarnings = FALSE)
```